doc/sphinx-guides/source/admin/dataverses-datasets.rst

```diff
@@ -33,6 +33,13 @@ Removes a link between a Dataverse collection and another Dataverse collection.
 
     curl -H "X-Dataverse-key: $API_TOKEN" -X DELETE http://$SERVER/api/dataverses/$linked-dataverse-alias/deleteLink/$linking-dataverse-alias
 
+List Dataverse Collection Links
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Provides information about whether a certain Dataverse collection ($dataverse-alias) is linked to or links to another collection. Only accessible to superusers. ::
+
+    curl -H X-Dataverse-key:$API_TOKEN http://$SERVER/api/dataverses/$dataverse-alias/links
```

Comment:
Thanks @philippconzett for the PR! The rest of the examples have quotation marks, so please add them here for consistency. I'm looking at some recent API calls that I've made to the Harvard Dataverse repository where I've used quotation marks and the commands have successfully run. Thanks again!